tox.ini

```diff
@@ -27,12 +30,6 @@ max-line-length=160
 [flake8]
 max-line-length=160
 
-[testenv:isort-check]
```

Comment:
I was not necessary to remove this from the tox/travis checks. Its a simple oversight, but you can simply update the `.isort.cfg` file and include the new third-party dependencies you have added to the project (jwt, werkzeug). After updating the config file run the tool again and commit any changes to the imports.

Reply:
Reverted changes to tox.ini and added dependencies to .isort.cfg.